src/Mod/PartDesign/Gui/TaskDressUpParameters.cpp

```diff
@@ -232,9 +237,28 @@ void TaskDressUpParameters::itemClickedTimeout() {
     wasDoubleClicked = false;
 }
 
+void TaskDressUpParameters::createSelectAllEdgesAction(QListWidget* parentList)
+{
+    // creates a context menu, a shortcut for it and connects it to e slot function
+
+    selectAllEdgesAction = new QAction(tr("Select all edges in the 3D view"), this);
+    selectAllEdgesAction->setShortcut(tr("Ctrl+Shift+A"));

```

Comment:
Do not use tr("Ctrl+Shift+A") because there is the risk that it won't be correctly translated and thus might break the shortcut. Instead use QKeySequence(QString::fromLatin1("Ctrl+Shift+A")) since Qt already provides the correct translation.